### DISCUSSION TITLE
ESP32 S3 , I WANT TO USE EXT  “Active oscillator”,Which registers do I need to modify？

### DISCUSSION THREAD

**Discussion by xia-qh-123** [micropython/micropython#15871] [ESP32]
ESP32 S3 HOW TO USE “Active oscillator”？Which registers do I need to modify？

**Comment by xia-qh-123** [comment]
<img width="735" alt="屏幕截图 2024-09-18 143148" src="https://github.com/user-attachments/assets/48966301-023b-45c7-a5d4-98f027516584">

**Comment by bidrohini** [comment]
You have to configure  RTC_CNTL_CLK_CONF_REG and RTC_CNTL_OPTIONS0_REG.
